src/components/AddPub.js

```diff
@@ -0,0 +1,329 @@
+import React, { Component } from "react";
```

Comment:
I can't comment on empty files, so this is here...

.ipynb_checkpoints should be deleted/gitignored